src/app/core/permissions/ability/ability.service.ts

```diff
@@ -42,15 +42,19 @@ export class AbilityService extends LatestEntityLoader<Config<DatabaseRules>> {
   }
 
   private updateAbilityWithUserRules(rules: DatabaseRules): Promise<any> {
-    const userRules = this.getRulesForUser(rules);
+    // If rules object is empty, everything is allowed
+    const userRules: DatabaseRule[] = rules
+      ? this.getRulesForUser(rules)
+      : [{ action: "manage", subject: "all" }];
```

Comment:
do we still need the separate init with all permissions then? (line 37)

Reply:
Yes, in case no rules were found at all this is relevant